apps/rwa/app/api/auth/[auth0]/route.ts

```diff
@@ -0,0 +1,12 @@
+import type { HandleAuth } from "@auth0/nextjs-auth0";
+import { handleAuth, handleLogin } from "@auth0/nextjs-auth0";
+
+// For some reason the inferred type of handleAuth is any so casting to HandleAuth
+export const GET = handleAuth({
+  login: handleLogin({
+    authorizationParams: {
+      scope: "openid profile read:all_data offline_access",
```

Comment:
Hard coding this instead of adding to `.env` file because this feels like config that defines how the application works rather than environment specific config.

In other words, if we change the values in the `.env` file then the application should work the same way just on different servers etc. But if we change this value then the application will behave differently.